libcutils/Makefile

```diff
@@ -22,6 +22,7 @@ commonSources := \
 	strdup8to16.c \
 	record_stream.c \
 	process_name.c \
+	properties.c \
```

Comment:
If you want me to cut-and-paste property_get_bool(), etc. I don't see how to reuse that without some changes to properties.c.